src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -641,7 +643,16 @@ public bool IsFrozenExplicitly
                 return false;
             }
         }
-
+        [Experimental("NVM_ISEXPERIMENTAL_GLPYH")]
```

Comment:
Does the glyph need to be experimental?

Reply:
No, but it has to be public AFAK for wpf bindig and was not sure we'd want to support it. Experimental experimental mode ;)